src/models/character/class-view.ts

```diff
@@ -0,0 +1,4 @@
+export interface ClassView { 
```

Comment:
This doesn't really have a direct connection to a class, so call it something else.